libraries/adaptive-expressions/package.json

```diff
@@ -32,6 +37,7 @@
     "big-integer": "^1.6.52",
     "@types/xmldom": "^0.1.34",
     "btoa-lite": "^1.0.0",
+    "eslint-plugin-only-warn": "^1.1.0",
```

Comment:
these should be as devDependencies

Reply:
the problem is with depcheck, maybe I could do what you're saying but I would need to skip this libraries in the depcheck analysis

Reply:
![image](https://github.com/user-attachments/assets/3c229267-ebd0-4b7d-b27d-638dce6d6c32)

Not so good idea

Reply:
For now, we can disable depcheck for that so it doesnt complain, either way, this is temporary, the only-warn will be removed after we fix each library